### DISCUSSION TITLE
Capture TTS speech with microphone from external speakers

### DISCUSSION THREAD

**Discussion by ghost** [open-webui/open-webui#6711] [General]
Hello, dear developers, thank you for your work.

Manjaro Linux, Doker, Firefox browser, all latest versions. Open WebUI Version
v0.3.35
  TTS Web API mode is selected. If I turn on the constant standby mode for voice input, then the last sentences of the message that is spoken by TTS are periodically captured. And these last sentences are transferred for input into a new message to the AI. I use a good microphone without echo or other glitches. And the computer speakers that stand nearby. 
Maybe it’s worth adding a voice activation phrase like in assistants, for example Okay Google.

**Comment by tjbck** [comment]
PR Welcome.
